app/assets/stylesheets/components/styled-box.scss

```diff
@@ -67,6 +67,12 @@
   }
 
   &__title {
+    width: 75%;
+    p { color: $color_font-light; }
```

Comment:
Would you consider styling this p tag with its own element per BEM practice? http://getbem.com/naming/

Reply:
I went back and forth on this because it just applies to a couple of paragraph tags, but went ahead and added a `styled-box__content-paragraph` to address this issue within the `__content` group (I searched through the codebase and there weren't any instances of a p tag inside `styled-box__title` so I just nixed this specific rule to tidy things up)